src/utils/data/animals.mock.json

```diff
@@ -26,19 +26,19 @@
     },
     {
       "id": "pipeline1.data_engineering",
-      "name": "Data Engineering"
+      "name": "Pipeline1 Data Engineering"
```

Comment:
@studioswong Does it make sense to send the name like this? Or should it still contains some sort of "level" separation. E.g. `Pipeline1 - Data Engineering`?

Reply:
If we're doing some sort of nesting we should use `.` or `/` syntax as this is analogous to a directory structure